Documentation/CodeSnippets/Manual/FieldInformation.rst.txt

```diff
@@ -0,0 +1,22 @@
+..  code-block:: php
```

Comment:
I would suggest to move this to a php file instead and include it via literalinclude. This way the content is linted and code styles are applied: https://docs.typo3.org/m/typo3/docs-how-to-document/main/en-us/Reference/ReStructuredText/Code/Codeblocks.html#literalinclude